<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemEffect</class>
 <widget class="QDialog" name="DialogSystemEffect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Effect...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-133</y>
          <width>626</width>
          <height>541</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="10" column="1" colspan="2">
            <widget class="QLabel" name="labelWithPrecision">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>with precision:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="15" column="3" colspan="4">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueScript" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QLabel" name="labelPercentVariance">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="3">
            <widget class="QRadioButton" name="radioButtonSpecialAction">
             <property name="text">
              <string>Special action:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="5" colspan="2">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>318</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="4">
            <widget class="QLabel" name="labelPercentCritical">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item row="7" column="4">
            <widget class="QLabel" name="labelPercentPrecision">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item row="11" column="4" rowspan="3" colspan="3">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>348</width>
               <height>89</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QCheckBox" name="checkBoxMaximum">
             <property name="text">
              <string>Maximum:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveMinimum" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0" rowspan="8">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>188</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="5" colspan="2">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>318</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueElementID" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="WidgetVariable" name="widgetVariableStockValueIn" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="4" colspan="3">
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="4" colspan="3">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>348</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="4" colspan="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>338</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="11" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueSkillAdd" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLabel" name="labelWithFormula">
             <property name="text">
              <string>with formula:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValuePrecision" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QCheckBox" name="checkBoxMinimum">
             <property name="text">
              <string>Minimum:</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QCheckBox" name="checkBoxElementID">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Element ID:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="2">
            <widget class="QCheckBox" name="checkBoxStockValueIn">
             <property name="text">
              <string>Stock value in:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButtonStatus">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QCheckBox" name="checkBoxCritical">
             <property name="text">
              <string>Critical:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueStatusPrecision" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueCritical" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButtonScript">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Script:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="4" colspan="3">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>348</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QCheckBox" name="checkBoxVariance">
             <property name="text">
              <string>Variance:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueStatusID" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QComboBox" name="comboBoxAddRemoveSkill">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Add</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Remove</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="3" colspan="3">
            <widget class="PanelDamagesKind" name="panelDamagesKind" native="true">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="10" column="4">
            <widget class="QLabel" name="labelPercentStatusPrecision">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item row="6" column="5" colspan="2">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>318</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="13" column="0" colspan="3">
            <widget class="QRadioButton" name="radioButtonCallCommonReaction">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Call common reaction:</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QRadioButton" name="radioButtonDamages">
             <property name="text">
              <string>Damages on </string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueDamageFormula" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="3">
            <widget class="QRadioButton" name="radioButtonPerformSkill">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Perform skill:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueVariance" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButtonAddRemoveSkill">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Skill:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QComboBox" name="comboBoxAddRemoveStatus">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Add</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Remove</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="12" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueSkillPerform" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="14" column="3" colspan="4">
            <widget class="QComboBox" name="comboBoxSpecialAction">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="4" colspan="3">
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QCheckBox" name="checkBoxPrecision">
             <property name="text">
              <string>Precision:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="5" colspan="2">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>318</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="3">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveMaximum" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="13" column="3">
            <widget class="QPushButton" name="pushButtonSelect_CallCommonReaction">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Select...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxTemporarilyChangeTarget">
         <property name="text">
          <string>Temporarily change target:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PanelPrimitiveValue" name="panelPrimitiveValueTemporarilyChangeTarget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelDamagesKind</class>
   <extends>QWidget</extends>
   <header>paneldamageskind.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetVariable</class>
   <extends>QWidget</extends>
   <header>widgetvariable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemEffect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemEffect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
